src/wazuh_modules/vulnerability_scanner/src/vulnerabilityScannerFacade.cpp

```diff
@@ -122,11 +128,10 @@ void VulnerabilityScannerFacade::start(
                 if (VerifyMessageBufferBuffer(verifier))
                 {
                     auto message = GetMessageBuffer(returnSlice.data());
-                    // If the message is newer than 60 second, wait to retry.
-                    auto secondsFromEpoch = std::chrono::duration_cast<std::chrono::seconds>(
-                                                std::chrono::system_clock::now().time_since_epoch())
-                                                .count();
-                    if ((message->timestamp() + SLEEP_RETRY_THREADS_SEC) < secondsFromEpoch)
+
+                    // If the message is newer than <SLEEP_RETRY_THREADS_SEC> second, wait to retry.
+                    const auto secondsFromEpoch = getSecondsFromEpoch();
```

Comment:
Capture getSecondsFromEpoch by copy, because the lifetime of the std::function getSecondsFromEpoch is closed to the start method.

Reply:
Done